### ISSUE TITLE
[Bug] [Linux] Bold text and emojis are not working

### ISSUE THREAD

**Issue by atraest** [etigerstudio/typora-misty-theme#15]
Hello. Nice theme! I just want to report some bugs I've found using the `misty-light-macos.css` and `misty-dark-macos.css` on Linux. I'm using Arch Linux, Kernel `5.6.3-arch1-1` on **i3-gaps** _Window Manager_.

- Bold text doesn't work.
- Emoji doesn't work.
- Other functions seems to work fine.

![bug](https://user-images.githubusercontent.com/57003922/79274887-a0ad2380-7e7b-11ea-92c2-890048f55a5b.png)

![bug-2](https://user-images.githubusercontent.com/57003922/79274895-a3a81400-7e7b-11ea-92b0-e985a71ba2b9.png)

**Comment by etigerstudio** [comment]
Glad to hear that you enjoyed this theme, while it's also a pity that certain markdown features are not properly worked as expected on your Arch Linux computer. I greatly appreciate your bug report for helping us identify possible defects of the theme.

## Bold Text
A quick fix might be just a switch from macOS versions to Windows, i.e. `misty-light-windows.css` or `misty-dark-windows.css`. Main reason behind this is these two versions use **different** font weights for bold text, and Windows version may be more appropriate for Linux distros.

*TL;DR: try `misty-light-windows.css`.*

## Emoji
Does emoji work in other Typora themes except Misty? If emoji rendering is broken **by this theme**, I'll look into the font fallback list of the theme to figure out some tweaks. Otherwise, you may check system-wise emoji support first.

*TL;DR: try `GitHub` theme that comes with Typora.*

Thanks for your report again. I'm waiting for your feedback so that I could spot the origin of those bugs & fix them eventually.

**Comment by atraest** [comment]
> Did emoji works in other Typora themes except Misty?

No. It worked before, weird.

> Otherwise, you may check system-wise emoji support first.

- I have no idea how to do this haha. I can try a clean install of Typora later I guess.
- The bold text problem only happens on your theme, works fine everywhere else.

> Thanks for your report again. I'm waiting for your feedback so that I could spot the origin of those bugs & fix them eventually.

No problem, glad to help.

**Comment by etigerstudio** [comment]
Thanks for your instant response.

> The bold text problem only happens on your theme, works fine everywhere else.

You could try downloading [Windows version of Misty](https://github.com/etigerstudio/typora-misty-theme/releases/download/v1.0-alpha.2/misty-light-windows.css) & switch to this file to see if bold text works.

> I have no idea how to do this haha. I can try a clean install of Typora later I guess.

"System-wise emoji support" simply means whether web browsers & text editors of your system render emoji correctly. `noto-fonts-emoji` package may give Arch Linux stable emoji support by the way.

**Comment by atraest** [comment]
> You could try downloading [Windows version of Misty](https://github.com/etigerstudio/typora-misty-theme/releases/download/v1.0-alpha.2/misty-light-windows.css) & switch to this file to see if bold text works.

It does! Now that we kinda nailed it, perhaps you should mention to Linux users on the `README.md` that they should use the Windows version for now. Otherwise you would have to fix the macOS one. I would suggest creating a similar version of Windows for Linux, if you need help testing you can count me in!

> "System-wise emoji support" simply means whether web browsers & text editors of your system render emoji correctly. `noto-fonts-emoji` package may give Arch Linux stable emoji support by the way.

Oh that definitely worked!

Thank you very much!

**Comment by etigerstudio** [comment]
I'm happy to know these issues have been solved.

> Now that we kinda nailed it, perhaps you should mention to Linux users on the README.md that they should use the Windows version for now. 

Currently, description on theme release page says:

> Download `misty-light-macos.css`🌞 or `misty-dark-macos.css`🌑 for macOS. Download `misty-light-windows.css`🌞 or `misty-dark-windows.css`🌑 for Windows & Linux (beta).

I have to admit I didn't make it clear enough. I'm considering a more direct way to show linux users which file should be downloaded, and hopefully this will come out in next release.

>  I would suggest creating a similar version of Windows for Linux, if you need help testing you can count me in!

You've done a great job as you thoroughly tested the theme & posted screenshots. Feel free to open another issue if you find other things that don't work well. I'm closing this issue.

Thanks.

**Comment by atraest** [comment]
> Download `misty-light-macos.css`sun_with_face or `misty-dark-macos.css`new_moon for macOS. Download `misty-light-windows.css`sun_with_face or `misty-dark-windows.css`new_moon for Windows & Linux (beta).

Oh I missed that, sorry.

> I have to admit I didn't make it clear enough. I'm considering a more direct way to show linux users which file should be downloaded, and hopefully this will come out in next release.

I appreciate that, thank you.

> You've done a great job as you thoroughly tested the theme & posted screenshots. Feel free to open another issue if you find other things that don't work well. I'm closing this issue.
> 
> Thanks.

No problem, happy to help. Will do.
